tests/simple/testdata/timestamps.textproto

```diff
@@ -325,7 +325,7 @@ section {
   }
   test {
     name: "get_milliseconds"
-    description: "Need to import a variable to get milliseconds."
+    description: "Obtain the milliseconds component of the duration. Note, this is not the same as converting the duration to milliseconds. This behavior will be deprecated."
     expr: "x.getMilliseconds()"
     type_env {
       name: "x"
```

Comment:
Can you change the `seconds: 123` below to `seconds: 321`? Took me a bit to realize this was only truncating the nanoseconds component to millis. Though to be fair as mentioned, this is not intuitive and will be removed at some point.

Reply:
No problem. Updated.